### PR TITLE
Document sending message to jupyterlite iframe

### PR DESCRIPTION
**Pull request by sebreb**
## References

An attempt to make a step-by-step tutorial that illustrate the feature described in #922.

## Code changes

Documentation: creation of a new how-to guide.

### CODE REVIEW

**Comment by jtpio** [comment]
Nice, many thanks @sebreb for putting that guide together!

File: docs/howto/configure/advanced/iframe.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+```bash
+cd examples
+python -m http.server -b 127.0.0.1
+```
+
+In a browser, at the address `http://127.0.0.1:8000`, you should be able to notice the
+communication between the host page and the IFrame (refresh the browser if necessary):
+
+https://user-images.githubusercontent.com/44410933/218273379-07d129e3-27c1-4c30-a89b-6d1a7bb61b26.mp4
```
Comment (jtpio) [inline docs/howto/configure/advanced/iframe.md:238]: Maybe we can convert this video to a gif so it renders on ReadTheDocs:

https://jupyterlite--980.org.readthedocs.build/en/980/howto/configure/advanced/iframe.html

**Comment by bollwyvl** [comment]
Of some interest, I recently hacked together [jyg](https://pypi.org/project/jyg/) which works with both full JupyterLab and  JupyterLite. 

I'm hoping to get a more integrated docs example e.g. a side-by-side scrolled document which runs commands in a notebook/console. 

The original impetus is to support controlling [jupyterlab-deck]() from [off-screen](https://github.com/deathbeds/jupyterlab-deck/issues/9). Consider an "advance slide" command that returns the speaker notes for that slide.

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by sebreb** [comment]
Happy to contribute! Thanks for the initial idea :)

**Pull request merged**